clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/config/DockerRegistryConfigurationProperties.groovy

```diff
@@ -53,6 +53,8 @@ class DockerRegistryConfigurationProperties {
     // or <library> for repositories like 'ubuntu'.
     List<String> repositories
     List<String> skip
+    // a file listing all targetd repositories
+    File catalogFile
```

Comment:
How are `File` classes serialized/deserialized? This might be better suited as a `String` and then converted

Reply:
Would we want to switch it up for `passwordFile` as well?